src/loader/aarch64/pe/mod.rs

```diff
@@ -0,0 +1,148 @@
+// Copyright (c) 2019 Intel Corporation. All rights reserved.
+// Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Copyright 2017 The Chromium OS Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE-BSD-3-Clause file.
+//
+// SPDX-License-Identifier: Apache-2.0 AND BSD-3-Clause
+
+//! Traits and structs for loading pe image kernels into guest memory.
+
+#![cfg(all(feature = "pe", target_arch = "aarch64"))]
```

Comment:
```suggestion
#![cfg(feature = "pe")]
```
The `aarch64` constraint is already gated by the `mod aarch64`.

Reply:
All suggested change applied.